backend/run/copr-backend-unknown-resalloc-tickets.py

```diff
@@ -33,7 +33,9 @@ def used_ids():
     output = output.strip()
     tickets = set()
     for ticket_id in output.split("\n"):
-        assert all(c.isdigit() for c in ticket_id)
+        if len(ticket_id) <= 0:
```

Comment:
This needs an explanation. A length of a string ticket ID, is a weird thing to calculate. Are you basically doing `if ticket_id:`?

Reply:
yes, updated